presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -370,7 +386,7 @@ public void tryStop()
         }
 
         try {
-            executor.run(this::stop);
+            executor.runAsync(this::stop).get();
```

Comment:
What is the difference between `run(...)` and  `runAsync(...).get()`?

Reply:
runAsync executed CheckedRunnable on scheduler

Reply:
?